AUTHORS.txt

```diff
@@ -58,6 +58,7 @@ Dimaslg <dimaslg@telecable.es>
 dixon13 <dixonbegay@gmail.com>
 Drill <drill87@gmail.com>
 Dudakov aka [OMCB]Kaban <dudakov.s@gmail.com>
+Drosfeh <drofseh@gmail.com>
```

Comment:
`sf` or `fs`? :)

Reply:
damn ... you have some good mistake tracking skillZ 🎉

Reply:
lol, it is `fs` thanks